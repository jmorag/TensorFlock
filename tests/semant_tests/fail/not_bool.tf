main = 12;

f : Bool -> Bool;
f b = if b then 32 else 12;
