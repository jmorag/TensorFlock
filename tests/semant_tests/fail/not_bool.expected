Fatal error: exception Failure("Declared type Bool -> Bool but received type Nat")
